webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = {
+  entry: path.resolve(__dirname, 'client', 'index.jsx'),
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'public'),
+  },
+  resolve: {
+    extensions: ['.js', '.jsx', '.json', '.css', '.scss'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        include: path.join(__dirname, 'client'),
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react'],
+          },
+        },
+      },
+    ],
+  },
+  mode: 'development',
```

Comment:
Nice job remembering to include this! (aside: webpack does default to development mode unless otherwise specified, though)

Reply:
Yeah, I just want to remember to change it:)